api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -528,4 +529,25 @@ message JwtAuthentication {
   // <http://www.w3.org/TR/cors/#cross-origin-request-with-preflight>`_ regardless of JWT
   // requirements specified in the rules.
   bool bypass_cors_preflight = 4;
+
+  // A map of unique requirement_names to JwtRequirements.
+  // :ref:`requirement_name <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.PerRouteConfig.requirement_name>`
+  // in `PerRouteConfig` uses this map to specify a JwtRequirement.
+  map<string, JwtRequirement> requirement_map = 5;
```

Comment:
should this be renamed to something like `per_route_requirements_definition` to make it more clear it's only used to define some JwtRequirement for per-route config and not other cases?

Reply:
My thinking is: this map could be used in other places beside per-route.   Actually,  there is another similar map in FilterStateRules.  But we deliberately called it FilterStateRules.  This one, I don't want to make the same mistake. I want to call it as general as possible.  In the future, so it can be used in other places.